deployment/proxies.rst

```diff
@@ -44,6 +45,10 @@ and what headers your reverse proxy uses to send information::
         // Request::HEADER_X_FORWARDED_AWS_ELB
     );
 
+.. versionadded:: 4.4
+
+    The support for the ``REMOTE_ADDR`` env var was introduced in Symfony 4.4.
```

Comment:
Again, not envvar, maybe 'Support for the ``'REMOTE_ADDR'`` value was introduced...'

Reply:
I have a question to better understand this feature in practice. Does it work like this?

* The user defines an env var called `REMOTE_ADDR` in ....... *(I don't know where)*
* You add the string `'REMOTE_ADDR'` here and Symfony will trust all the proxies defined in the `REMOTE_ADDR` env var

Is this correct? Thanks.

Reply:
Nope, see https://www.php.net/manual/en/reserved.variables.server.php

Reply:
No :)

`$_SERVER['REMOTE_ADDR']` is a value set by PHP with the IP address of the computer making the request to the server.

When you pass the string value 'REMOTE_ADDR' to setTrustedProxies it is replaced with the value of `$_SERVER['REMOTE_ADDR']`. Its the same as calling `setTrustedProxies($_SERVER['REMOTE_ADDR'])`. This doesnt look very helpful, but keep going...

Env vars come into this because we can make that call be `setTrustedProxies($_ENV['TRUSTED_PROXIES'])`. So now we can configure trusted proxies via the env - allowing us to, for example, have different trusted proxies for dev and prod.

And we set the env var `TRUSTED_PROXIES=REMOTE_ADDR`.

Reply:
Thanks for your detailed explanation. Sadly I still don't understand anything 😓  ... but it's my fault, so don't worry. I've reworded this. Does it look better to you? Thanks